src/test/java/org/symphonyoss/integration/webhook/jira/parser/v2/CommentMetadataParserTest.java

```diff
@@ -107,7 +107,6 @@ public void testRestrictedComment() throws IOException, JiraParserException {
   }
 
   @Test
-  @Ignore("This is only valid when supporting mentions in the metadata")
```

Comment:
You might remove the verify at the end of the method.

You should also change the userService mocking statement to avoid anyString().

Something like that:
doReturn(user).when(userService).getUserByUserName("mockUser", integrationUser);